docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Einstein for Developers Setup
+lang: en
+---
+
+## Set Up Overview
+
+A Salesforce admin or user with the appropriate permissions can install Einstein for Developers in a supported Salesforce org. A developer with access to the org can then:
+
+1.  Install the extension in VS Code.
+2.  Connect to the org to use the extension.
+
+### Required Salesforce Editions and User Permissions
+
+**Available in**: Developer, Enterprise, Partner Developer, Performance and Unlimited Editions.
+
+**Inoperable in**: Group, Professional and Essentials Editions
+
+### User Permissions Needed
+
+**Configure Einstein for Developers(beta)**: `CustomizeApplication` and `ModifyAllData`
+
+## Enable Einstein for Developers
+
+An Admin can enable Einstein for Developers in a supported Salesforce org by following the given steps.
+
+1. From Setup, enter Einstein for Developers in the Quick Find box, then select Einstein for Developers. If you don’t see Einstein for Developers as an option under Setup in your Developer Edition org, it was likely created before Einstein for Developers was introduced and doesn’t have the correct permissions. To continue, create another Developer Edition org.
+2. Enable Einstein for Developers, and then review and accept the license agreement.
```

Comment:
Can we put extra emphasize here that we encourage their admin to thoroughly review the license agreement *before* accepting the terms and conditions and enabling Einstein for Developers? The modal links out to the T&Cs so it'd be good to draw their attention to that

Reply:
done.